src/cf.data.pre

```diff
@@ -2418,6 +2418,30 @@ DOC_START
 			The proxy_protocol_access is required to whitelist
 			downstream proxies which can be trusted.
 
+	   worker-queues
+			Ask TCP stack to maintain a dedicated listening queue
+			for each worker accepting requests at this port.
+			Requires TCP stack that supports the SO_REUSEPORT socket
+			option.
+
+			Cons: Enabling worker-specific queues allows any process
+			running as Squid's effective user to accept requests
+			destined to this port! Also, a stuck worker is assigned
+			the same number of new connections as active workers,
+			inconveniencing more users compared to the single shared
+			queue (default) configuration where a blocked worker
+			cannot affect new connections (as long as there are
+			other, active workers running).
```

Comment:
@eduard-bagdasaryan, good point!

Reply:
Does that mean you plan to remove the config settings that let such caches be used by SMP workers sometime very soon? Until then it is an annoying effect the admin using such (bad) configurations should be made aware of.

If you wish to state this in the wiki instead of the config docs that would also be fine by me.

Reply:
> Does that mean you plan to remove the config settings that let such caches be used by SMP workers sometime very soon?

I am not aware of such plans. I would not recommend _prohibiting_ such unsupported configurations, would welcome adding startup warnings about them, but that discussion is out of scope here.

> Until then it is an annoying effect the admin using such (bad) configurations should be made aware of.

I agree, but this option is not introducing any new annoying effects. The same annoyances existed before and will continue to exist after this PR, in configurations with or without this option. We should not be discussing these annoyances here.

> lists of 'pros vs cons' is not something typically done in the config file anyway

@eduard-bagdasaryan, I suggest removing these cons and pros paragraphs to avoid wasting time on addressing this change request. IMO, these paragraphs provide valuable information to the admin, but this review is making them too expensive to keep. Just leave the security warning paragraph that Amos insists on in another change request.

Reply:
The change seems not to be done, or not made it to github yet. These two review items should auto-close by github when the relevant lines are actually removed.